client/influxdb.go

```diff
@@ -252,11 +274,11 @@ func (a Results) Error() error {
 // Precision can be specified if the timestamp is in epoch format (integer).
 // Valid values for Precision are n, u, ms, s, m, and h
 type Point struct {
-	Name      string
-	Tags      map[string]string
-	Timestamp time.Time
-	Fields    map[string]interface{}
-	Precision string
+	Name      string                 `json:"name,omitempty"`
```

Comment:
It is important for a couple of reasons:

1) What does adding them do to change the code?  In this case, nothing, as we aren't using these tags anywhere in the code.
2) Leaving them there will imply to a future developer they are in play, but they are not.


There is no need to notate that the tags don't omitempty with time as it isn't being used here now.